javascript-forms/form-events.js

```diff
@@ -0,0 +1,31 @@
+function handleFocus(event) {
+  console.log("'focus' event was fired");
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log("'blur' event was fired");
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleInput(event) {
+  var name = event.target.name;
+  var value = event.target.value;
+  console.log('value of', name, ':', value);
+}
+
+var $name = document.forms[0][0];
+var $email = document.forms[0][1];
+var $message = document.forms[0][2];
```

Comment:
You should still use `querySelector` here, this may break if another form is ever added to the page